### ISSUE TITLE
TypeError('decoding to str: need a bytes-like object, _io.BufferedReader found',)

### ISSUE THREAD

**Issue by pdehaan** [pdehaan/speaktome-loadtests#8]
When running the tests locally on Mac using `-cxvv`, we see the following error. Not sure if this is with my code, or something from AIOHTTP/Molotov.

```
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Access-Control-Allow-Origin: *
Content-Encoding: gzip
Content-Type: application/json; charset=utf-8
Date: Thu, 27 Jul 2017 23:33:02 GMT
ETag: W/"6b3-OFkn7MqBabPXrYGIhLzQfRJCdFI"
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Powered-By: Express
Content-Length: 283
Connection: keep-alive

**** Binary content ****

<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

Task exception was never retrieved
future: <Task finished coro=<LoggedClientSession.print_request() done, defined at /Users/pdehaan/dev/github/mozilla/speaktome-loadtests/.tox/osx/lib/python3.6/site-packages/molotov/session.py:80> exception=TypeError('decoding to str: need a bytes-like object, _io.BufferedReader found',)>
Traceback (most recent call last):
  File "/Users/pdehaan/dev/github/mozilla/speaktome-loadtests/.tox/osx/lib/python3.6/site-packages/molotov/session.py", line 101, in print_request
    body = str(body, 'utf8')
TypeError: decoding to str: need a bytes-like object, _io.BufferedReader found
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

**Comment by pdehaan** [comment]
@tarekziade Any ideas? This is baffling.

**Comment by pdehaan** [comment]
Tests pass, just oddly dumps that out for every test execution.

**Comment by tarekziade** [comment]
This is a Molotov bug. Fixing it

**Comment by tarekziade** [comment]
Fixed in Molotov 1.3 (released on PyPI) 

Thanks for the feedback and digging

**Comment by pdehaan** [comment]
Closing. Was fixed via new molotov@1.3 release.
